EvidenceApi.Tests/V1/UseCase/FindResidentsBySearchQueryUseCaseTests.cs

```diff
@@ -60,7 +62,7 @@ public void ReturnsTheResidentByResidentNameAndTeam()
             var result = _classUnderTest.Execute(request);
 
             // Assert
-            result.Count.Should().Be(1);
+            result.Count.Should().Be(2);
```

Comment:
maybe adding a third resident with different parameters that is not returned..? just to show it's correctly excluded since this was how the test used to work